sympy/calculus/util.py

```diff
@@ -569,6 +570,89 @@ def lcim(numbers):
 
     return result
 
+def log_convex(func, *syms, **kwargs):
+    """Checks if the function passed in the argument is logarithamically convex.
+    If the function is logarithamically convex then it return `True`, otherwise,
+    `False`.
+    To check for strict logarithmic convexity, `strict = True` can be used.
+
+    Examples
+    ========
+
+    >>> from sympy import symbols, exp, log_convex
+    >>> x = symbols('x', real = True, positive = True)
+    >>> log_convex(exp(x), x, strict = True)
+    False
+    >>> log_convex(x**(-1), x)
+    True
+
+    References
+    ==========
+
+    .. [1] https://en.wikipedia.org/wiki/Logarithmically_convex_function
+    """
+
+    if len(syms) > 1:
+        raise NotImplementedError("Log convexity check for multivariate functions is not implemented yet.")
+
+    func = _sympify(func)
+    strictness = kwargs.get('strict', False)
+    var = syms[0]
+    upper_bound = 1
+    if func.diff(var) == 0:
+        ratio = func * func.diff(var).diff(var)
+        upper_bound = 0
+    else:
+        ratio = func * func.diff(var).diff(var) / (func.diff(var)**2)
+    if strictness:
+        comp = ratio <= upper_bound
+    else:
+        comp = ratio < upper_bound
+    if solve_univariate_inequality(comp, var):
+        return False
+    return True
+
+def log_concave(func, *syms, **kwargs):
```

Comment:
The logic of these two functions is basically identical. I would simply leave this function's docstring, but remove the code and have it just call `log_convex(-func)`.